simSPI/linear_simulator/shift_utils.py

```diff
@@ -0,0 +1,90 @@
+"""Module containing method to shift an image in Fourier domain."""
+import numpy as np
+import torch
+
+
+class Shift(torch.nn.Module):
+    """class containing method to shift an image in Fourier domain.
+
+    Written by H. Gupta and Y.S.G. Nashed.
+
+    Parameters
+    ----------
+    config: class
+        Class containing parameters of the shift
+
+    """
+
+    def __init__(self, config):
+        super(Shift, self).__init__()
+
+        self.config = config
+        self.frequency = 1.0 / (self.config.side_len * self.config.pixel_size)
+
+        n2 = float(self.config.side_len // 2)
```

Comment:
I think `n_half_len` would be more explicit. `n2` could suggest nsquared for example.